CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Changed
 
+- Changed the element of `IconMenu` to our `Box` component ([@InstaK](https://github.com/InstaK) in [#324](https://github.com/teamleadercrm/ui/pull/324))
```

Comment:
PP: a bit more clear wording:

- `IconMenu` now returns a `Box` component instead of a `div`